README.md

```diff
@@ -26,22 +26,51 @@ gem install modulr-api
 
 ## Usage
 
-TODO: Write usage instructions here
-
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `bundle exec rake` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+
+To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then tag and push the new version:
+
+```git
+git tag vx.x.x main
+git push origin vx.x.x
+```
+
+The tagging will trigger the GitHub action defined in `release.yml`, pushing the gem to [rubygems.org](https://rubygems.org).
+
+## Tooling
+
+We have a tooling system based on HTTP request files which allow developers to run pre-format Modulr API requests easily. These HTTP request files are stored inside `doc/modulr_requests` directory. Find the service you want to run and run the specific HTTP file to execute the request locally.
+
+We run these requests locally from our favorite IDLEs, in our case, we use ([IntelliJ](https://www.jetbrains.com/es-es/idea/) or [VS Code](https://code.visualstudio.com/)). [IntelliJ](https://www.jetbrains.com/es-es/idea/) has a native feature that allows you to run these requests easily but if you are using [VS Code](https://code.visualstudio.com/) you will need to install the following extension [httpYac](https://marketplace.visualstudio.com/items?itemName=anweber.vscode-httpyac).
```

Comment:
I am missing a brief explanation of how to configure the HTTP files environment. (the `http-client.private.env.json` file)